media/css/pocket/components/updates-signup.scss

```diff
@@ -24,10 +30,95 @@ $image-path: '/media/protocol/img';
 
     p {
         @include text-body-lg;
+    }
 
-        a {
-            color: $color-action-primary;
+    .mzp-c-button {
+        background-color: $color-button-red;
+        border: 0;
+        height: 40px;
+
+        &:hover,
+        &:focus,
+        &:active {
+            background-color: color.adjust($color-button-red, $lightness: -10%);
+            color: $color-white;
         }
     }
+}
+
+    .pocket-updates-form {
+        .mzp-u-list-styled {
+            margin: 8px;
+            padding: 0;
+
+            li {
+                margin: 0;
+            }
+        }
+
+        fieldset {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+
+            label {
+                @include visually-hidden;
+            }
+
+            @media (min-width: 870px) {
+                flex-direction: row;
+
+                input[type="email"] {
+                    flex-grow: 2;
+                    margin-right: 30px;
+                    margin-bottom: 0;
+                }
+            }
+        }
+
+        .mzp-c-form-errors {
+            max-width: $content-md;
+
+            &:last-child {
+               margin: 0 auto $spacing-lg;
+            }
+        }
 
+        @media #{$mq-lg} {
+            .mzp-c-form-errors {
+                &:last-child {
+                    margin-bottom: $spacing-2xl;
+                }
+            }
+        }
+
+    }
+
+.legal-links {
+    display: flex;
+    justify-content: center;
+
+    p {
+        @include text-body-sm;
+    }
+
+    p:first-of-type {
+        margin-right: $spacing-xs;
+    }
+
+    p:last-of-type {
+        margin-left: $spacing-xs;
+    }
+
+    .separator {
+        @include text-body-md;
+    }
+
+    a {
+        color: $color-black;
+    }
+
+    @media (min-width: 870px) {
```

Comment:
Question: Is this number specific to this page alone, or is ti something we can / should standardize as a common break point?

Reply:
Have bumped this to 960px to deal with the overflow issue spotted above - can't answer the broader question I'm afraid - one for @reemhamz when she's back, but not a blocker I think?